src/interfaces/apv.ts

```diff
@@ -1,4 +1,5 @@
 export interface ISimpleApv {
+  raw: string;
   version: number;
   extra: { [key: string]: string };
```

Comment:
If we know certain keys we have we can list them here.

```typescript
export interface ISimpleApv {
  raw: string;
  version: number;
  extra: {
    player: string;
    launcher: string;
    [key: string]: unknown;
  };
}
```

Reply:
Cool